releasenotes/notes/bedrock-support-bce28e3078c85c12.yaml

```diff
@@ -0,0 +1,17 @@
+---
+prelude: >
+    Amazon Bedrock models are now supported in Haystack. All currently available
+    models are supported, including the ones that are already announced
+    (e.g. Llamma-2-70b-chat). The models can be used in the same way as other
+    models in Haystack using its model ids as `model_name_or_path` (see
+    https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids-arns.html).
+
```

Comment:
How about:
Haystack now supports Amazon Bedrock models, including all existing and previously announced models, like Llamma-2-70b-chat. To use these models, simply pass the model ID in the `model_name_or_path` parameter, like you do for any other model. For details, see [Amazon Bedrock Docmentation](https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids-arns.html).

Reply:
small typo: it's Llama-2-70b-chat